Assets/MRTK/SDK/Features/Utilities/Migration/Tools/MigrationTool.cs

```diff
@@ -326,12 +378,16 @@ private bool MigrateGameObjectHierarchy(GameObject parent)
                     {
                         changedAnyGameObject = true;
                         migrationHandlerInstance.Migrate(child.gameObject);
-                        Debug.Log($"Successfully migrated {parent.name} object");
+
+                        status.log += $"Successfully migrated {child.gameObject.name} object \n";
+                        Debug.Log(status.log);
                     }
                 }
                 catch (Exception e)
                 {
-                    Debug.LogError($"{e.Message}: GameObject {parent.name} could not be migrated");
+                    status.failures++;
+                    status.log += $"{e.Message}: GameObject {child.gameObject.name} could not be migrated \n";
+                    Debug.LogError(status.log);
```

Comment:
If we encounter 5 errors, I think this will lead to an output of 15 different lines, where the first error is logged 5 times total, the second one is logged 4 times total, and so on.

Appending the status to the log is good, but maybe we should only output the stuff at the end so that we don't multi-log